File: corpus/2017q1/_p2q3_ok/_check/F;0600.java

import java.util.Scanner;
public class questao3 {

    public static void main(String[] args) {

        Scanner teclado = new Scanner (System.in);

    int L, C, i, j;

    System.out.println("Digite dimensão para na linha L: ");
    L=teclado.nextInt();

    System.out.println("Digite Coluna C:");
    C=teclado.nextInt();

    float matrizA[][];
    matrizA = new float [L][C];

    for(i=0;i<L;i++){
    for(j=0; j<C; j++){
    System.out.println("Digite o elemento da linha "+(j+1)+" coluna "+(i+1)+":");
    matrizA[i][j]=teclado.nextFloat();
  }
 }

    for(i=0;i<L;i++){
    for(j=0; j<C; j++){
    System.out.println(matrizA[i][j]);
    }
    }

     for (i=0;i<5;i++){
    System.out.println(matriz[i][n]);
    }

}

File: corpus/2017q1/_p2q3_ok/_check/F;0600.py
Scanner teclado = new Scanner (System.in)
int L, C, i, j
print("Digite dimensão para na linha L: ")
L=int(input())
print("Digite Coluna C:")
C=int(input())
float matrizA[][]
for i=0 in range(0, L):
    for j=0 in range(0, C):
        print("Digite o elemento da linha "+(j +1)+" coluna "+(i +1)+":")
for i=0 in range(0, L):
    for j=0 in range(0, C):
        print(matrizA[i][j])
for i=0 in range(0, 5):
    print(matriz[i][n])